### ISSUE TITLE
Broken Equation Rendering

### ISSUE THREAD

**Issue by stefnotch** [Marc-B-Reynolds/Marc-B-Reynolds.github.io#6]
@Marc-B-Reynolds The (probably quite pretty) equations on http://marc-b-reynolds.github.io/math/2017/04/18/TaitEuler.html#error don't show up for me. For example

![image](https://user-images.githubusercontent.com/10220080/96490064-9a3fa000-1240-11eb-87cf-cc546d0e9c48.png)

**Comment by Marc-B-Reynolds** [comment]
Yeah. github pages updated the versions of some tools which broke all my equation rendering.  Thanks for reporting.
